GliaWidgets/Component/OnHoldOverlay/OnHoldOverlayStyle.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+public struct OnHoldOverlayStyle {
```

Comment:
Since we introduce a new style, I'd like to propose to avoid extending the global namespace and introduce this structure inside of Theme namespace for example.
What do you think, does it make sense?
cc: @igorkravchenko

Reply:
This is consitent to what we have with other components. While I don't mind it being in Theme generally, I think consistency would win here until we would refactor the themeing

Reply:
@dukhovnyi @kuldar-daniel I think because most of the component styles are declared globally in the module, it makes sense to do the same for this style.

Reply:
ok, let's do this way